llvm-external-projects/iree-dialects/test/iree_pydm/to_iree/structural.mlir

```diff
@@ -109,3 +124,25 @@ iree_pydm.func @raise_on_failure_builtin(%arg0 : !iree_pydm.exception_result, %a
   raise_on_failure %arg0 : !iree_pydm.exception_result
   return %arg1 : !iree_pydm.integer
 }
+
+// -----
+// CHECK-LABEL: @call_and_visibility
+iree_pydm.func @call_and_visibility(%arg0 : !iree_pydm.integer) -> (!iree_pydm.exception_result, !iree_pydm.integer) {
+  // CHECK: %[[R:.*]]:2 = call @callee(%arg0) : (i32) -> (i32, i32)
+  %0:2 = call @callee(%arg0) : (!iree_pydm.integer) -> (!iree_pydm.exception_result, !iree_pydm.integer)
```

Comment:
nice - and IPO hoisting of constant return values will really help all this melt away even further - the IR is going to look a lot more like a hand transcription of the python than something produced by a transpiler!